src/main/java/mate/academy/onlinebookstoreproject/service/impl/BookServiceImpl.java

```diff
@@ -7,15 +7,16 @@
 import mate.academy.onlinebookstoreproject.dto.CreateBookRequestDto;
 import mate.academy.onlinebookstoreproject.mapper.BookMapper;
 import mate.academy.onlinebookstoreproject.model.Book;
-import mate.academy.onlinebookstoreproject.repository.impl.BookRepositoryImpl;
+import mate.academy.onlinebookstoreproject.repository.BookRepository;
 import mate.academy.onlinebookstoreproject.service.BookService;
 import org.springframework.stereotype.Service;
 
 @RequiredArgsConstructor
 @Service
 public class BookServiceImpl implements BookService {
     private static final String CANNOT_FIND_BOOK_BY_ID_MSG = "Cannot find book by Id: ";
-    private final BookRepositoryImpl bookRepository;
+    private static final String FAILED_TO_UPDATE_BY_ID_MSG = "There is book with the provided ID: ";

```

Comment:
For me looks like incorrect message. Please check it one more time and think about better one. I would use CANNOT_FIND_BOOK_BY_ID_MSG as the cause is the same, book is not found by ID.